Ownership/TC - LTC/London TC DAGGA.txt

```diff
@@ -22,7 +22,7 @@ ALTOWNER:Observing LON_C_CTR:DAGGA:REDFA:TCE:LE:O:LM:LC:LNC:LWC:LSC:L
 ALTOWNER:Observing LON_E_CTR:DAGGA:REDFA:TCE:O:LM:LE:LC:LNC:LWC:LSC:L
 ALTOWNER:Observing LTC_E_CTR:DAGGA:REDFA:O:LM:TCE:LE:LC:LNC:LWC:LSC:L
 ALTOWNER:Observing London FIR:O:LM:DAGGA:REDFA:TCE:LE:LC:LNC:LWC:LSC:L
-BORDER:LTC2:LTC3a:LTC3b:LTC4:LTC5
```

Comment:
These two sub-sectors can be replaced by one single sub-sector:

```
SECTOR:London TC DAGGA 2 (155-245):15500:24500
OWNER:DAGGA:REDFA:TCE:LE:LC:LNC:LWC:LSC:L
ALTOWNER:Observing LON_SC_CTR:DAGGA:REDFA:TCE:LE:LC:LNC:LWC:O:LM:LSC:L
ALTOWNER:Observing LON_C_CTR:DAGGA:REDFA:TCE:LE:O:LM:LC:LNC:LWC:LSC:L
ALTOWNER:Observing LON_E_CTR:DAGGA:REDFA:TCE:O:LM:LE:LC:LNC:LWC:LSC:L
ALTOWNER:Observing LTC_E_CTR:DAGGA:REDFA:O:LM:TCE:LE:LC:LNC:LWC:LSC:L
ALTOWNER:Observing London FIR:O:LM:DAGGA:REDFA:TCE:LE:LC:LNC:LWC:LSC:L
BORDER:LTC2:LTC3a:LTC3b:LTC34b:LTC33a:LTC36:LTC4:LTC5
```

![image](https://user-images.githubusercontent.com/12017284/71619955-f7d60380-2bbe-11ea-8147-86757b8d8c83.png)